server.js

```diff
@@ -326,6 +326,11 @@ var formidable = require('formidable'); // Multiple file upload library
 var cheerio = require('cheerio');
 var request = require('request');
 
+// use the cors cross origin REST model
+webServer.use(cors());
```

Comment:
The old webServer.use(cors()) later on in this file should be deleted so it isn't redundant

Reply:
Good catch, I had originally moved it up to the new position but I assume a merge put it back at the original spot at some point